pkg/controller/hostpathprovisioner/contoller_test.go

```diff
@@ -120,15 +124,15 @@ var _ = Describe("Controller reconcile loop", func() {
 			},
 		}
 		saNN := types.NamespacedName{
-			Name:      "test-name-admin",
+			Name:      "hostpath-provisioner-admin",
```

Comment:
Use the other const you just created in common.go

Reply:
Done